lms/djangoapps/instructor_task/tasks_helper/grades.py

```diff
@@ -297,9 +302,38 @@ def grouper(iterable, chunk_size=self.USER_BATCH_SIZE, fillvalue=None):
             args = [iter(iterable)] * chunk_size
             return izip_longest(*args, fillvalue=fillvalue)
 
-        users = CourseEnrollment.objects.users_enrolled_in(context.course_id, include_inactive=True)
-        users = users.select_related('profile')
-        return grouper(users)
+        def users_for_course(course_id):
```

Comment:
Is there a reason to have both `users_for_course` & `users_for_course_v2` outside of the waffle toggle? If yes, at the minimum there is a comment for the `v2` version of this method noting that it is by chunk, can we add a brief comment here to describe that is getting users all at once? If no, can we denote that one will go away once the waffle is removed?

Reply:
Sure, I should add it.